packages/monorepo-builder/packages/init/src/Composer/PackageNameVersionProvider.php

```diff
@@ -31,8 +32,7 @@ public function provide(string $packageName): string
         $version = null;
 
         try {
-            $prettyVersion = PrettyVersions::getVersion($packageName)
-                ->getPrettyVersion();
+            $prettyVersion = $this->getPrettyVersion($packageName);
 
```

Comment:
This class should be re-used in the calls above. To avoid duplication.

Reply:
I'm not sure how to do that, since I don't know enough about the app... I should inject this class in the other ones, but I don't know who's doing the injection...

Reply:
We use constructor injection here